docs/tutorials/how-to-write-contracts-that-talk-to-each-other.md

```diff
@@ -167,7 +149,7 @@ describe("Calculator", function() {
 
   // Common setup below
   beforeAll(async function () {
-    near = await nearApi.connect(nearConfig);
+    near = await nearlib.connect(nearConfig);
```

Comment:
You'll need to use the new pattern found here for example: https://github.com/near-examples/guest-book/blob/master/src/index.js